service/co/hotwax/configuration/CompanyConfigurationServices.xml

```diff
@@ -0,0 +1,279 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<services xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/service-definition-3.xsd">
+
+    <service verb="create" noun="UserWithOrganization" authenticate="anonymous-all">
```

Comment:
We should see this as two business process, as system will allow to create new user for existing organisation.